### DISCUSSION TITLE
PCAP action does not exist in the actions menu

### DISCUSSION THREAD

**Discussion by foxman0719** [Security-Onion-Solutions/securityonion#13987] [2.4] [answered]
### Version

2.4.110

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

16

### RAM

64

### Storage for /

1T

### Storage for /nsm

8T

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

I want to analyze packets in the PCAP option but they do not appear and I cannot send them from the actions menu because the option is not there.

![Captura de pantalla 2024-11-27 150941](https://github.com/user-attachments/assets/fa7c09ae-7dfe-4bf8-a43c-55fa04808134)
![Captura de pantalla 2024-11-27 151129](https://github.com/user-attachments/assets/8ef0c199-1fae-4f1b-a18d-ef37ca950e10)

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by dougburks** [comment, marked as answer]
Looking at your first screenshot, it appears you are looking at the default Alerts view which is `Grouped by Name, Module`. If you select the Drilldown option on the context menu, then it should show you the individual alerts themselves. If you then click on an individual alert, you should see the PCAP option on the context menu.

From https://docs.securityonion.net/en/2.4/alerts.html#actions:
> Clicking the PCAP option will pivot to the [PCAP](https://docs.securityonion.net/en/2.4/pcap.html#pcap) interface to retrieve full packet capture for the selected stream. This option will only appear if you click on a log that contains source IP, source port, destination IP, destination port, etc.

**Reply by foxman0719** [reply]
is ok, thanks for your help
